src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_validators.py

```diff
@@ -573,7 +585,7 @@ def _validate_vm_create_nics(namespace):
     namespace.public_ip_type = None
 
 
-def _validate_vm_create_auth(namespace):
+def _validate_create_auth(namespace):
```

Comment:
Same comment. I recommend leaving the "vm" in here.  The format here is _validate_{command name}_{param}.  Hence, since it's the auth for "vm create", that's what it was named the way it was.